pkg/cli/config/config.go

```diff
@@ -24,13 +25,14 @@ func GetBackplaneConfigFile() string {
 
 // Get Backplane config default path
 func getConfiDefaultPath(fileName string) string {
-	configDir, err := os.UserConfigDir()
-
+	UserHomeDir, err := os.UserHomeDir()
 	if err != nil {
 		return fileName
 	}
 
-	return configDir + "/" + fileName
+	configFilePath := filepath.Join(UserHomeDir, info.BACKPLANE_CONFIG_DEFAULT_FILE_PATH, fileName)
```

Comment:
Since we're pulling the `info` package in here to get the data directly out, we might as well do that for both the default filename and the default config path.  Thoughts?
```suggestion
	configFilePath := filepath.Join(UserHomeDir, info.BACKPLANE_CONFIG_DEFAULT_FILE_PATH, info.BACKPLANE_CONFIG_DEFAULT_FILE_NAME)
```